website/content/docs/plugins/containerized-plugins.mdx

```diff
@@ -0,0 +1,253 @@
+---
+layout: docs
+page_title: Containerized Plugins (Beta)
+description: Running external plugins for Vault in containers.
+---
+
+# Containerized Plugins (Beta)
```

Comment:
I'm not sure if we usually include the beta in the page title, but more as a note in the body?

Reply:
```suggestion
# Containerized plugins (Beta)
```